api/StargateDocv2.yaml

```diff
@@ -1,920 +1,2100 @@
-openapi: 3.0.1
+---
+openapi: 3.0.3
 info:
- title: Stargate Document API reference
- description: "The Stargate Document API provides CRUD operations on document data managed by Stargate."
- license:
-   name: Apache 2.0
-   url: http://www.apache.org/licenses/LICENSE-2.0.html
- version: 2.0.0
+  title: Stargate Document API
+  description: "The Stargate Document API provides CRUD operations on document data managed by Stargate."
+  termsOfService: https://www.datastax.com/legal
+  contact:
+    name: DataStax
+    url: https://www.datastax.com/contact-us
+  version: 2.0.13
 servers:
- - url: https://localhost:8082
-   description: Replace the port if necessary.
+- url: /api/docs
```

Comment:
The server returned /api/docs? If it returns /api/rest we should leave that for now.

Reply:
Yes, the server actually returned /api/rest for the latest Document API swagger, so ok I'll change it back to that.  Thanks.

Reply:
Done in latest commit.